### PR TITLE
fix a typo at the singlediode function

### LINKED ISSUE #2177: Typo
**Describe the bug**
Typo in word, `arugments`

**To Reproduce**
https://github.com/pvlib/pvlib-python/blob/b5e89be2dc7ec761fd1bf6eb21691fc10590c95b/pvlib/singlediode.py#L635

**Expected behavior**
To read `arguments`

**Versions:**
 - ``pvlib.__version__``: 0.11.0

> [!WARNING]  
**THIS ISSUE IS RESERVED FOR THE 2024 EU PVPMC Workshop.
PLEASE DO NOT WORK ON THIS ISSUE WITHOUT PRIOR AGREEMENT.**

### PR DESCRIPTION
**Pull request by luizreiscver**
- [X] Closes #2177 
 - [X] I am familiar with the [contributing guidelines](https://pvlib-python.readthedocs.io/en/latest/contributing.html)
 - [X] Adds description and name entries in the appropriate "what's new" file in [`docs/sphinx/source/whatsnew`](https://github.com/pvlib/pvlib-python/tree/main/docs/sphinx/source/whatsnew) for all changes. Includes link to the GitHub Issue with `` :issue:`num` `` or this Pull Request with `` :pull:`num` ``. Includes contributor name and/or GitHub username (link with `` :ghuser:`user` ``).
 - [X] New code is fully documented. Includes [numpydoc](https://numpydoc.readthedocs.io/en/latest/format.html) compliant docstrings, examples, and comments where necessary.
 - [X] Pull request is nearly complete and ready for detailed review.
 - [x] Maintainer: Appropriate GitHub Labels (including `remote-data`) and Milestone are assigned to the Pull Request and linked Issue.

### CODE REVIEW

**Comment by echedey-ls** [comment]
Can any maintainer please approve workflows when possible? TIA.

File: pvlib/singlediode.py
Comment (mikofski) [inline pvlib/singlediode.py:635]: Remove trailing white space please

**Review by cwhanse — APPROVED** [review]

**Comment by kandersolar** [comment]
Thanks @luizreiscver!  We hope to see more contributions from you :)

**Pull request merged**
